### PR TITLE
DRILL-7484: Malware found in the Drill test folder

### PR DESCRIPTION
**Pull request by cgivre**
This PR replaces the `attack-trace.pcap` file with `http.pcap` and rewrites accordingly the unit tests for the TCP Sessionization feature.

### CODE REVIEW

File: exec/java-exec/src/test/java/org/apache/drill/exec/store/pcap/TestSessionizePCAP.java [hunk 1]
```diff
@@ -47,61 +55,129 @@ public static void setup() throws Exception {
...
-    String sql = "SELECT COUNT(*) FROM cp.`/store/pcap/attack-trace.pcap`";
+    String sql = "SELECT COUNT(*) FROM cp.`/store/pcap/http.pcap`";
     String plan = queryBuilder().sql(sql).explainJson();
     long cnt = queryBuilder().physical(plan).singletonLong();
-    assertEquals("Counts should match", 5L, cnt);
+    assertEquals("Counts should match", 1L, cnt);
+  }
+
+
+  private static String readAFileIntoString(String filePath)
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/test/java/org/apache/drill/exec/store/pcap/TestSessionizePCAP.java]: This method has numerous code style issues:

- `{` are on the new lines
- extra spaces before `(`, ``)`
- `e.printStackTrace();`

Comment (cgivre) [reply exec/java-exec/src/test/java/org/apache/drill/exec/store/pcap/TestSessionizePCAP.java]: Cleaned up this function.

**Comment by cgivre** [comment]
1.  It looks like you added the colon.
2.  I tested `http.pcap` on VirusTotal [1] and it checked out with no malicious content.
3.  I added the file `dataFromRemote.txt` which contains an http response.  The reason was that the content would have been a REALLY long string, so I figured it would be best to put that in a separate file, rather than in the unit test itself.  

[1] https://www.virustotal.com/gui/file/25a72bdf10339f2c29916920c8b9501d294923108de8f29b19aba7cc001ab60d/detection

Screen Shot 2019-12-15 at 9 52 29 AM

**Comment by arina-ielchiieva** [comment]
+1

**Comment by cgivre** [comment]
Thanks all for the review.  Commits squashed.

**Pull request closed without merging**